weblate/templates/manage/backups.html

```diff
@@ -45,7 +45,7 @@ <h4 class="panel-title">
             <button type="button" class="btn btn-link btn-xs" data-clipboard-text="{{ service.passphrase }}" title="{% trans "Copy to clipboard" %}"><i class="fa fa-copy"></i></button>
         </td></tr>
         <tr><th>{% trans "SSH key" %}</th><td>
-            <a href="{% url 'manage-ssh-key' %}" class="btn btn-primary">{% trans "Download private key for backup" %}</a>
+            <a href="{% url 'manage-ssh-key' %}" class="btn btn-primary">{% trans "Download private key for performing backup" %}</a>
```

Comment:
It's more ... for backing up the key, not for performing the backup. It would be actually used for restoring the backup if needed.